privacyidea/static/components/user/views/user.add.html

```diff
@@ -1,5 +1,15 @@
+<style type="text/css">
+  .css-form input.ng-invalid.ng-touched {
+    background-color: #FA787E;
+  }
+
+  .css-form input.ng-valid.ng-touched {
+    background-color: #78FA89;
+  }
+</style>
+
 <div class="panel" ng-controller="userAddController">
-    <form name="formAddUser" role="form" validate>
+    <form name="User" role="form" novalidate>
```

Comment:
Now I believe if memory serves I changed the form name so I could modify the form from the controller. Unfortunately Angular is a newish technology to me so I suspect I just didn't/don't understand how to do what I wanted with the old form name.